src/state/sagas/app/FetchInitialData/FetchInitialData.js

```diff
@@ -33,6 +34,16 @@ export function * fetchAllInitialData (action) {
     const scenarioTree = yield select(selectFirstScenarioFromScenarioTree);
     if (scenarioTree.length !== 0) {
       yield call(fetchScenarioByIdForInitialData, workspaceId, scenarioTree[0].id);
+      // Start state polling for running scenarios
+      for (let i = 0; i < scenarioTree.length; ++i) {
+        if (scenarioTree[i].state === 'Running') {
```

Comment:
Could you please extract scenario state returned by BO (Running, Failed, Created,...) ?